scripts/systemtap/kem/kemd.c

```diff
@@ -55,9 +54,9 @@ static void kemd_cleanup_module(void);
 static int kemd_open(struct inode *inode, struct file *filp);
 static int kemd_release(struct inode *inode, struct file *filp);
 static ssize_t kemd_read(struct file *filp, char *buf,
-			 size_t len, loff_t *offset);
+                         size_t len, loff_t *offset);
```

Comment:
We prefer the original code.

The web browser has some issues to show the correct indent.
Please note:
* We use TAB for indentation.
* We prefer to align the variables.